Octokit.Reactive/Helpers/ConnectionExtensions.cs

```diff
@@ -28,6 +28,15 @@ public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connectio
             return GetPages(url, parameters, (pageUrl, pageParams) => connection.Get<List<T>>(pageUrl, pageParams, null).ToObservable());
         }
 
+        public static IObservable<T> GetAndFlattenAllPages<T>(this IConnection connection, Uri url, IDictionary<string, string> parameters, ApiOptions options)
```

Comment:
obviuosly you've needed more overloads here for calls that use dictionary of parameters and also ApiOptions (and also it looks like we may yet have another requirement for a `GetAll` call that needs to take `ApiOptions` and uses  an `accepts` header.

I wonder if there is a cleaner way to rationalise all the overloads here to just call through to one implementation method, passing default/empty parameters for the dictionary and acceptheader etc...  rather than having the pagination setup call etc in more than one place.
@shiftkey can advise on the approach here